.github/workflows/update-internal-base-images.yml

```diff
@@ -24,8 +24,8 @@ jobs:
             git config --global user.name "auto dockerfiles update"
             echo "==== $(date): Starting pipenv setup... ===="
             python -m pip install --upgrade pip
-            pip install pipenv==2023.3.18
-            pipenv install
+            pip install pipenv==2024.4.0
```

Comment:
do we need to specify a version at all?

Reply:
Yes, because we're testing it on a specific version and if tomorrow there will be a new version for pipenv with BCs it will be broken again.